### ISSUE TITLE
Inner element of progress bar referenced using aria-labelledby is not recognised as progress bar name

### ISSUE THREAD

**Issue by adamsamec** [nvaccess/nvda#13912]
### Steps to reproduce:
1. Visit [this demo page][example].
2. Make sure that browse mode of NvDA is enabled.
3. Press the "1" key to jump to the 'Accessibility tests of the "progressbar" ARIA role' heading.
4. Read through the page using the Down arrow key until you reach the "End of examples" button.

### Actual behavior:
Below each of the &lt;h2&gt; heading, is an example element with the "progressbar" role, where each progressbar is named using a different method. When focused using the browse mode, the accessible name of the example 1 progressbar, the "Updating..." string, is announced as expected. However, in the example 2 progressbar, the accessible name, which should be determined from the progressbar's inner element referenced using the "aria-labelledby" attribute, is not announced.

### Expected behavior:
The textual content of the element inside of the example 2 progressbar, the "Updating..." string, should be announced as the progressbar's accessible name. Clearly, the "aria-labelledby" reference to the inner element is ignored, as if the inner element would not have existed. The example 3 progressbar demonstrates that the problem is not in the usage  of the "aria-labelledby" attribute for accessible name computation, as in this case, where the referenced element lies outside of the progressbar, the accessible name is determined correctly.

The issue with the example 2 progressbar is quite serious, as this case is most likely the one which the web authors would use often in the real case.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
NVDA 2022.1.

#### Windows version:
Windows 11, version 21H2 (OS Build 22000.795)

#### Name and version of other software in use when reproducing the issue:
Google Chrome, version 103.0.5060.114

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

[example]: https://jsbin.com/vifuxuheta/edit?html,output

**Comment by seanbudd** [comment]
This looks like a duplicate of #13307, can you confirm?
Let us know if it's not a duplicate so we can reopen this issue.

**Comment by adamsamec** [comment]
Yes, looks like it is a more general problem of ignoring the label when the element referenced by "aria-label" is inside the element referencing that inner element.
